ServiceTier/Services/PresetService.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Common.ModelsDTO;
+using Timer.DAL.Extensions;
+using Timer.DAL.Timer.DAL.Entities;
+using Timer.DAL.Timer.DAL.UnitOfWork;
+
+namespace ServiceTier.Services
+{
+    public class PresetService : BaseService, IPresetService
+    {
+        public PresetService(IUnitOfWork unitOfWork) : base(unitOfWork)
+        {
+
+        }
+
+        public void CreatePreset(PresetDTO presetDTO)
+        {
+            Preset preset = presetDTO.ToPreset();
+            unitOfWork.Presets.Create(preset);
+            unitOfWork.Save();
+            var presets = unitOfWork.Presets.GetAll();
+            foreach (var timer in presetDTO.Timers)
+            {
+                timer.PresetId = preset.Id;
+                unitOfWork.Timers.Create(timer.ToTimer());
+                unitOfWork.Save();
+            }
+        }
+
+        public void UpdatePreset(PresetDTO presetDTO)
+        {
+            Preset preset = presetDTO.ToPreset();
+            unitOfWork.Presets.Update(preset);
+            unitOfWork.Save();
+        }
+
+        public void UpdateTimer(TimerDTO timerDTO)
```

Comment:
this feels a little bit out of place here. how about moving it to it's own 'timer' service?